### PR TITLE
#161697250 Configure eslint

### PR DESCRIPTION
**Pull request by obulaworld**
#### What does this PR do?
 - Setup eslint

#### Description of Task to be completed?
- setup eslint
- configure eslint

#### How should this be manually tested?
- start the app by running ```yarn start:dev```
the app should open on port ```8080``` on your browser

#### What are the relevant pivotal tracker stories?
[#161697250](https://www.pivotaltracker.com/story/show/161697250, "Configure eslint")

### CODE REVIEW

File: src/components/AuthenticatedRoute.js [hunk 1]
```diff
@@ -4,23 +4,23 @@ import React from 'react';
 // third party library
 import { Route, Redirect } from 'react-router-dom';
 
-//Moduler Routes
+// Moduler Routes
 import Auth from '../helpers/TokenCheck';
 
 /**
  * desc renders header with links
```
Comment (KaiserPhemi) [inline src/components/AuthenticatedRoute.js]: This should actually read 
```js
@desc
```

File: src/components/Router.js [hunk 2]
```diff
@@ -2,13 +2,13 @@
 import React from 'react';
 
 // third party library
-import { Switch, Route, Router  } from 'react-router-dom';
+import { Switch, Route, Router } from 'react-router-dom';
```
Comment (KaiserPhemi) [inline src/components/Router.js:5]: #### Nitpick
When commenting imports it's actually one of the following (though this is not a comprehensive list)
```js
// react libraries
...react packages

// third-party libraries 
....other installed packages

// components
....components 

// actions
...


```

So, remove the Moduler importations stated on line 7 and in every other file.

File: src/index.js [hunk 3]
```diff
@@ -1,15 +1,20 @@
...
-  <AppRouter />,
-  document.getElementById('root')
-);
+const store = configureStore();
+
+ReactDOM.render((
+    <Provider store={store}>
+    <AppRouter />
+    </Provider>),
+   document.getElementById('root'));
```
Comment (KaiserPhemi) [inline src/index.js]: #### Nitpick
No new line here.

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]

File: src/components/Router.js (hunk 2 above)
Comment (obulaworld) [reply src/components/Router.js:5]: Okay. On it

Comment (obulaworld) [reply src/components/Router.js:5]: Fixed

**Review by KaiserPhemi — APPROVED** [review]

**Pull request merged**